pkg/utils/constants.go

```diff
@@ -71,6 +71,7 @@ var MissingOrIncorrectSecondaryDNS = CustomError{Code: 32, Message: "MissingOrIn
 var InvalidParameterCombination = CustomError{Code: 33, Message: "InvalidParameterCombination"}
 var FailedReadingConfiguration = CustomError{Code: 34, Message: "FailedReadingConfiguration"}
 var MissingOrInvalidConfiguration = CustomError{Code: 35, Message: "MissingOrInvalidConfiguration"}
+var PasswordsDoNotMatch = CustomError{Code: 36, Message: "PasswordsDoNotMatch"}
```

Comment:
This needs to be a different Code, we are already using 36 for InvalidUserInput. I'd say make it 38